docs/how-to-guides/index.md

```diff
@@ -14,6 +14,7 @@
 - [Advanced usage of colcon](others/advanced-usage-of-colcon.md)
 - [Applying Clang-Tidy to ROS packages](others/applying-clang-tidy-to-ros-packages.md)
 - [Using Eagleye in Autoware](others/eagleye-integration-guide.md)
+- [Important temporal performance metrics on components](others/important-temporal-performance-metrics.md)
```

Comment:
```suggestion
- [Important temporal performance metrics on components](others/defining-temporal-performance-metrics.md)
```

Reply:
CI found a deadlink due to change in the file name. Please fix this, and I can merge this PR